src/components/PopupSimple.js

```diff
@@ -0,0 +1,25 @@
+import Popup from "./Popup.js";
+
+export default class PopupSimple extends Popup {
```

Comment:
`PopupSimple` should be called `PopupWithConfirmation` according to the check list